e2e/test-component/scenarios/embedding-sdk/dashboard-click-behavior.cy.spec.tsx

```diff
@@ -300,23 +305,26 @@ describe("scenarios > embedding-sdk > dashboard-click-behavior", () => {
               },
             ],
           }).then(({ dashboard }) => {
+            stubAnchorClick();
+
             mountSdkContent(
               <InteractiveDashboard dashboardId={dashboard.id} />,
             );
 
             getSdkRoot().within(() => {
               H.getDashboardCard(0)
-                .findByRole("link", {
-                  name: "https://example.org/448",
-                })
-                .should("have.attr", "href", "https://example.org/448");
+                .findAllByText("https://example.org/761")
```

Comment:
I did change some of the tests to not always open the same link, this is just to make sure that the tests are passing because they're correct and not because the spies are not cleared across tests